src/relay/ir/op.cc

```diff
@@ -95,6 +95,17 @@ const bool Op::HasGenericAttr(const std::string& key) {
   return true;
 }
 
+// Resets attr of the OpMap.
+void OpRegistry::reset_attr(const std::string& key) {
+  OpManager* mgr = OpManager::Global();
+  std::lock_guard<std::mutex> lock(mgr->mutex);
+  std::unique_ptr<GenericOpMap>& op_map = mgr->attr[key];
+  if (op_map == nullptr) {
+    return;
+  }
+  op_map->data_.clear();
```

Comment:
per https://docs.tvm.ai/contribute/code_review.html#ensure-test-coverage

Need testcase for reset_attr.
The current implementation resets the attr of other ops along the current one, which is not correct.

Reply:
I see. I will take a look. I have misunderstood the registry codebase then.